addons/hr_timesheet/models/project_task.py

```diff
@@ -124,7 +124,10 @@ def _get_group_pattern(self):
         }
 
     def _get_groups_patterns(self):
-        return ['(?:%s)*' % self._get_group_pattern()['planned_hours']] + super()._get_groups_patterns()
+        res = super()._get_groups_patterns()
```

Comment:
for me

{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "python",
            "request": "launch",
            "python": "/usr/bin/python3",
            "program": "odoo/odoo-bin",
            "args": [
                "--addons-path=odoo/addons/,enterprise/",
                "--http-port=8069",
                "--database=shortcut",
                "--test-file=odoo/addons/hr_timesheet/tests/test_project_task_quick_create.py",
                "--limit-time-cpu=0",
                "--limit-time-real=0",
                "--limit-request=0",
            ],
            "cwd": "${workspaceRoot}",
            "console": "integratedTerminal",
        },
    ]
}